compiler/test/compilable/fix24187.c

```diff
@@ -2,7 +2,7 @@
 
 // https://issues.dlang.org/show_bug.cgi?id=24187
 
-#ifdef linux
+#ifdef __linux__
```

Comment:
Ah okay, so that's what made `-std=gnu11` required with the GNU cpp?

Reply:
Basically, yes:
```
[happy@happypc] ~ % gcc -E -std=gnu11 -dD -xc /dev/null | grep linux
#define __gnu_linux__ 1
#define __linux 1
#define __linux__ 1
#define linux 1
[happy@happypc] ~ % gcc -E -std=c11 -dD -xc /dev/null | grep linux
#define __gnu_linux__ 1
#define __linux 1
#define __linux__ 1
```